yb-voyager/cmd/exportData.go

```diff
@@ -170,6 +171,29 @@ func exportData() bool {
 			log.Errorf("Failed to prepare dbzm config: %v", err)
 			return false
 		}
+		if source.DBType == POSTGRESQL && changeStreamingIsEnabled(exportType) {
```

Comment:
The second clause is redundant because of the condition on line 168.

Reply:
Technically no, because line 168 tests `if changeStreamingIsEnabled(exportType) || useDebezium` which also kicks in when we have BETA_FAST_DATA_EXPORT=1 in offline migration :D (in which case we  want to use debezium)

But it's worth rethinking -  we'd still want to use pg_dump when some uses BETA_FAST_DATA_EXPORT with pg because it's the faster option?